manifests/server.pp

```diff
@@ -56,7 +52,6 @@
   }
 
   if $clamav  { include '::clamav' }
```

Comment:
Doesn't the same problem apply to inclusion of clamav, auditd and pam?  Seems like the inclusion of these classes is already controlled by data-in-modules....